app/controllers/companies_controller.rb

```diff
@@ -23,4 +23,10 @@ def update_companies_details
     flash[:notice] = 'Companies details successfully updated'
     redirect_to companies_url
   end
+
+  def update_company_shareholders
+    CompanyShareholdersUpdator.call(params[:isin])
```

Comment:
Don't you want to require it?

```suggestion
    CompanyShareholdersUpdator.call(params.require(:isin))
```

Reply:
Yes , I should require `:isin` here